File: TitusWarehouse/warehouse-2.0.1/server/AirSysco.Packaging/PackageSet.vb
Imports System
Imports System.Text

Public Class PackageSet
    Public Sub PackageSet(ByVal Weight As Decimal, ByVal length As Int32, ByVal Width As Int32, ByVal Height As Int32)
        weightLB = Weight
        lengthIN = length
        widthIN = Width
        heightIN = Height
    End Sub
    Public weightLB As Decimal
    Public lengthIN As Int32
    Public widthIN As Int32
    Public heightIN As Int32
End Class

File: TitusWarehouse/warehouse-2.0.1/server/AerSysCo.Packaging/PackageSet.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace AerSysCo.Packaging
{
public class PackageSet {
    public PackageSet(Decimal weight, int length, int width, int height) {
        weightLB = weight;
        lengthIN = length;
        widthIN = width;
        heightIN = height;
    }

    public Decimal weightLB;
    public int     lengthIN;
    public int     widthIN;
    public int     heightIN;
}
}
